Main.java

```diff
@@ -2,20 +2,30 @@
   public static void main(String[] args) {
    // kindly use appropriate data types for your declaration
    // declare variables days the debtor defaulted, amount the debtor is to pay per day,amount the debtor paid and assign values to them
-   
+        int defaultedDays= 5;
+        int amountToPayPerDay= 1000;
+        int amountPaid= 3000;
    //Declare the variable of the total amount the debtor is to pay, the days the debtor paid for, amount the debtor is left to pay, days the debtor has not payed for
-
+        int amountToPay;
+        int daysPaidFor;
+        int daysToPay;
+        int amountLeftToPay;
+        int daysNotCaptured;
```

Comment:
Good job but your amounts should have been in float, and you should have used camelCase naming method to name your variables